pkg/workloads/console/acceptance/acceptance.go

```diff
@@ -261,12 +257,56 @@ func (r *Runner) Run(logger kitlog.Logger, config *rest.Config) {
 				return err
 			}).ShouldNot(HaveOccurred(), "could not find authorisation")
 
-			// TODO: Check that job has not been created
-			// TODO: Check that console phase is currently 'Pending'
+			By("Expect the console phase is pending authorisation")
+			Eventually(func() workloadsv1alpha1.ConsolePhase {
+				console, err = client.WorkloadsV1Alpha1().Consoles(namespace).Get(consoleName, metav1.GetOptions{})
+				Expect(err).NotTo(HaveOccurred(), "could not find console")
+				return console.Status.Phase
+			}).Should(Equal(workloadsv1alpha1.ConsolePendingAuthorisation))
+
+			By("Expect that the job has not been created")
+			Eventually(func() error {
+				_, err = client.BatchV1().Jobs(namespace).Get(consoleName, metav1.GetOptions{})
+				return err
+			}).Should(HaveOccurred(), "expected not to find job, but did")
+
+			By("Expect a console authorisation has been created")
+			consoleAuthorisation := &workloadsv1alpha1.ConsoleAuthorisation{}
+			Eventually(func() error {
+				consoleAuthorisation, err = client.WorkloadsV1Alpha1().ConsoleAuthorisations(namespace).Get(consoleName, metav1.GetOptions{})
+				return err
+			}).ShouldNot(HaveOccurred(), "could not find console authorisation")
+
+			// Change the console user to another user as a user cannot authorise their own console
+			By("Update the console user")
+			console.Spec.User = "another-user@example.com"
+			console, err = client.WorkloadsV1Alpha1().Consoles(namespace).Update(console)
+			Expect(err).NotTo(HaveOccurred(), "could not update console user")
 
-			// TODO: Add an authorisation
+			By("Authorise a console")
+			consoleAuthorisation.Spec.Authorisations = []rbacv1.Subject{{Kind: "User", Name: user}}
+			_, err = client.WorkloadsV1Alpha1().ConsoleAuthorisations(namespace).Update(consoleAuthorisation)
+			Expect(err).NotTo(HaveOccurred(), "could not authorise console")
 
-			// TODO: Check that job has now been created
+			By("Expect a job has been created")
+			Eventually(func() error {
+				_, err = client.BatchV1().Jobs(namespace).Get(jobName, metav1.GetOptions{})
+				return err
+			}).ShouldNot(HaveOccurred(), "could not find job")
+
+			By("Expect a pod has been created")
+			Eventually(func() ([]corev1.Pod, error) {
+				opts := metav1.ListOptions{LabelSelector: "job-name=" + jobName}
+				podList, err := client.CoreV1().Pods(namespace).List(opts)
+				return podList.Items, err
+			}).Should(HaveLen(1), "expected to find a single pod")
+
+			By("Expect the console phase is Running")
+			Eventually(func() workloadsv1alpha1.ConsolePhase {
+				console, err = client.WorkloadsV1Alpha1().Consoles(namespace).Get(consoleName, metav1.GetOptions{})
+				Expect(err).NotTo(HaveOccurred(), "could not find console")
+				return console.Status.Phase
+			}).Should(Equal(workloadsv1alpha1.ConsoleRunning))
```

Comment:
Arguably these two checks could be dropped, as they're duplicating what we have in the main consoles happy-path test. No harm leaving them in if they're quick to execute though.

Reply:
Yeah this is true, I was just excited to see this working!

Will remove them as once the job is created we are all good!

Reply:
Actually going top merge this with the happy path test. Everything else is covered in integration tests.